monitor/linuxmonitor/processor.go

```diff
@@ -159,36 +112,28 @@ func (s *LinuxProcessor) Start(eventInfo *rpcmonitor.EventInfo) error {
 // Stop handles a stop event
 func (s *LinuxProcessor) Stop(eventInfo *rpcmonitor.EventInfo) error {
 
-	contextID, err := generateContextID(eventInfo)
-	if err != nil {
-		return fmt.Errorf("Couldn't generate a contextID: %s", err)
-	}
-
-	if !strings.HasPrefix(contextID, cgnetcls.TriremeBasePath) || contextID == cgnetcls.TriremeBasePath {
-		return nil
+	contextID := eventInfo.PUID
+	if len(eventInfo.Cgroup) > 0 {
+		if !s.regStop.Match([]byte(eventInfo.Cgroup)) {
+			return fmt.Errorf("Invalid PUID")
```

Comment:
same. Especially that you match a `eventInfo.Cgroup` and complain `Invalid PUID` :)